packages/gradle/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://gradle.org/
 TERMUX_PKG_DESCRIPTION="Powerful build system for the JVM"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:7.2
+TERMUX_PKG_VERSION=7.3
```

Comment:
Why are you removing package epoch? It isn't supposed to be removed even while updating the package

Reply:
Changes committed. Now its ok?